src/ring_theory/witt_vector/isocrystal.lean

```diff
@@ -176,8 +176,8 @@ begin
     intros ha',
     apply this,
     simp only [←ha, ha', zero_smul] },
-  obtain ⟨b, hb, m, (hmb : φ(p, k) b * a = p ^ m * b)⟩ :=
-    witt_vector.exists_frobenius_solution_fraction_ring p ha,
+  obtain ⟨b, hb, m, hmb⟩ := witt_vector.exists_frobenius_solution_fraction_ring p ha,
```

Comment:
I think a better fix here would be to redefine `witt_vector.exists_frobenius_solution_fraction_ring` to be stated in terms of `fraction_ring.frobenius`

Reply:
@eric-wieser, I think given this is just a "speed-up-a-proof" PR, we can just ping @hrmacbeth as the author of this file, and leave this proposal for later?